SignInCheckIn/SignInCheckIn.Tests/Services/ChildSigninServiceTest.cs

```diff
@@ -803,10 +803,10 @@ public void CheckAcEventDuringCurrentService()
             // Arrange
             _applicationConfiguration.Setup(m => m.AdventureClubEventTypeId).Returns(20);
 
-            DateTime currentMpEventDtoStartTime = new DateTime(2016, 12, 1, 9, 0, 0);
-            DateTime futureMpEventDtoStartTime = new DateTime(2016, 12, 1, 11, 0, 0);
+            var currentMpEventDtoStartTime = new DateTime(2016, 12, 1, 9, 0, 0);
```

Comment:
In this particular case at least var vs none var did not cause any issues.  I think we should investigate the issue if the test is failing due to var as we use that throughout the code and if it is causing issues in tests we should be concerned with the code outside of tests having the same problem.  In the case you mentioned my guess is if we had specified `new System.List<T>` var would not have caused and issue.